zk_toolbox/crates/zk_inception/src/consts.rs

```diff
@@ -6,3 +6,4 @@ pub const PROVER_MIGRATIONS: &str = "prover/prover_dal/migrations";
 pub const PROVER_STORE_MAX_RETRIES: u16 = 10;
 pub const DEFAULT_CREDENTIALS_FILE: &str = "~/.config/gcloud/application_default_credentials.json";
 pub const DEFAULT_PROOF_STORE_DIR: &str = "artifacts";
+pub const BELLMAN_CUDA_DIR: &str = "era-bellman-cuda";
```

Comment:
could we add this as a param for ecosystem config?

Reply:
Definitely 👍